notebooks/ImvizExample.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "17b4af6f",
```

Comment:
Note: I have to regenerate the cell IDs because Lab is acting weird on me with cells not rendering.

Reply:
Sorry about this. If this `id` stuff is too annoying to review, I can try to revert the changes.

Reply:
Didn't quite fix the problem (jupyterlab/jupyterlab#11078) but too late to turn back now. 😢

Reply:
Nah this is fine.  I agree it's annoying, but I've been seeing it in other places too.  Probably something to live with until jupyterlab responds :shrug: